### DISCUSSION TITLE
Meaning of `go depth 18`

### DISCUSSION THREAD

**Discussion by fishchess** [official-stockfish/Stockfish#4247] [Q&A]
What is meaning of `go depth 18`  ?

Does Stockfish calculate at least **_ply 18_** ?

How many moves are calculate to the _depth 18_ ?

**Comment by XInTheDark** [comment]
`go depth 18` means calculate moves until depth 18.

**Comment by XInTheDark** [comment]
> Does Stockfish calculate at least **_ply 18_** ?

Yes, in fact it will calculate exactly until depth 18.

> How many moves are calculate to the depth 18 ?

It should be displayed under `nodes searched` I believe

**Comment by Sopel97** [comment]
More precisely, it instructs stockfish to search the position for 18 iterations, where each subsequent iteration is more thorough than the previous. There is no simple connection to depth.

**Comment by fishchess** [comment]
> More precisely, it instructs stockfish to search the position for 18 iterations,

17 iterations are not important and only the last iteration is important ... 

I want to know the difference between depth 12 and depth 18?

**Reply by vondele** [reply]
There is no simple answer to your question the depth you talk about  is what is called the rootdepth. The actual depth reached in search can be very different from that. See the information and graph in https://github.com/official-stockfish/Stockfish/discussions/3888 

The average depth searched will increase roughly linearly with the rootDepth required, that's roughy it. The strength of the engine clealy increases.
